package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./index.html",
   "scripts": {
     "fix": "standard --fix",
-    "serve": "live-server --port=3000 --host=localhost --no-css-inject",
+    "serve": "live-server --port=8081 --host=localhost --no-css-inject",
```

Comment:
Why the port change?

Reply:
The Google Maps API only works on 8081, this is why we changed port number